### PR TITLE
Font faces

### PR DESCRIPTION
**Pull request by Starzu**

### CODE REVIEW

File: core/src/main/scala/scalacss/Css.scala [hunk 1]
```diff
@@ -53,14 +58,16 @@ object Css {
   type ValuesByMediaQuery = NonEmptyVector[(CssSelector, NonEmptyVector[CssKV])]
   type ByMediaQuery       = Map[CssMediaQueryO, ValuesByMediaQuery]
 
-  def findStylesAndAnimations(c: Css): (Stream[CssStyleEntry], Stream[CssKeyframesAnimation]) = {
+  def findStylesAndAnimationsAndFontFaces(c: Css): (Stream[CssStyleEntry], Stream[CssKeyframesAnimation], Stream[CssFontFace]) = {
```
Comment (grzesiul) [inline core/src/main/scala/scalacss/Css.scala]: can that be decoupled into (if that makes sense):
- findSyles
- findAnimations
- findFontFaces
- findAllTheStuff
  ?

File: core/src/main/scala/scalacss/package.scala [hunk 2]
```diff
@@ -56,6 +60,7 @@ package object scalacss {
                            content: NonEmptyVector[CssKV]) extends CssEntry
   case class CssKeyframesAnimation(name: KeyframeAnimationName,
                                    frames: Map[KeyframeAnimationSelector, Stream[CssStyleEntry]]) extends CssEntry
+  case class CssFontFace(fontFamily: String, src: Seq[String], fontStretch: Value, fontStyle: Value, fontWeight: Value, unicodeRange: UnicodeRange) extends CssEntry
```
Comment (grzesiul) [inline core/src/main/scala/scalacss/package.scala:63]: Hope CSSEntry is sealed trait so all match'es are checked

File: core/src/main/scala/scalacss/Css.scala (hunk 1 above)
Comment (Starzu) [inline core/src/main/scala/scalacss/Css.scala]: There is no need to decouple it, but I will change this name.

File: core/src/main/scala/scalacss/package.scala (hunk 2 above)
Comment (Starzu) [inline core/src/main/scala/scalacss/package.scala:63]: Yes, CSSEntry is sealed trait.

**Pull request merged**